sleap/gui/commands.py

```diff
@@ -2995,9 +2995,20 @@ def set_visible_nodes(
             if node in copy_instance and not copy_instance[node].isnan():
                 # just copy x, y, and visible
                 # we don't want to copy a PredictedPoint or score attribute
+                x_old = copy_instance[node].x 
+                y_old = copy_instance[node].y 
+                old_size_width=  copy_instance.frame.video.shape[2]
+                old_size_height=  copy_instance.frame.video.shape[1]
+                new_size_width=  new_instance.frame.video.shape[2]
+                new_size_height=  new_instance.frame.video.shape[1]
+                x_new = (x_old/old_size_width) * new_size_width
+                y_new = (y_old/old_size_height) * new_size_height
```

Comment:
Similarly, we can probably optimize by calculating (outside the for loop) a single factor to multiply by as 
```python
scale_width = new_size_width / old_size_width
```
same for height and then
```suggestion
                x_new = x_old * scale_width
                y_new = y_old * scale_height
```